src/main/java/de/themoep/inventorygui/InventoryGui.java

```diff
@@ -1316,7 +1316,7 @@ public void onInventoryDrag(InventoryDragEvent event) {
                 // Check if we only drag over one slot if so then handle that as a click with the element
                 if (event.getRawSlots().size() == 1) {
                     int slot = event.getRawSlots().iterator().next();
-                    if (slot < event.getView().getTopInventory().getSize()) {
+                    if (slot < GuiView.of(event.getView()).getTopInventory().getSize()) {
```

Comment:
If an exception occurs on the reflection this event should be cancelled here too.